tig_stack/bolt-project.yaml

```diff
@@ -1,3 +1,9 @@
 name: tig
 plans:
 - tig 
+
+modules:
+  - puppet-grafana
+  - quadriq-influxdb
+  - puppetlabs-stdlib
```

Comment:
```suggestion
```

This will be added when modules are resolved, since it's a dependency of the other 3.

Reply:
Oh good catch, I think I just copy pastad from the old Puppetfile 🙈